--- /frontend/src/lib/api/core/OpenAPI.ts	2023-06-05 18:28:25.613524100 +0000
+++ /frontend/src/lib/api/core/OpenAPI.ts 2023-06-05 18:29:05.197524100 +0000
@@ -19,7 +19,7 @@
 };

 export const OpenAPI: OpenAPIConfig = {
-    BASE: '',
+    BASE: import.meta.env.VITE_isWeb ? `https://${import.meta.env.VITE_CSNAME}-8000.preview.app.github.dev` : 'http://127.0.0.1:8000',
     VERSION: '0.1.0',
     WITH_CREDENTIALS: false,
     CREDENTIALS: 'include',
